cinema/serializers.py

```diff
@@ -111,10 +149,18 @@
     taken_places = TicketSeatsSerializer(
         source="tickets", many=True, read_only=True
     )
+    movie_image = MovieImageSerializer(many=False, read_only=True)
```

Comment:
Similarly, in `MovieSessionDetailSerializer`, the `movie_image` field is using `MovieImageSerializer`. Ensure this is intended, as it might expect a nested object rather than just the image field.